schemas/README.md

```diff
@@ -992,6 +1455,122 @@ Line number in the file
 </tr>
 </table>
 
+## ModelPrimitive
+
+A primitive representing a 3D model file loaded from an external URL
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>pose</code></td>
+<td>
+
+[Pose](#pose)
+
+</td>
+<td>
+
+Origin of model relative to reference frame
+
+</td>
+</tr>
+<tr>
+<td><code>scale</code></td>
+<td>
+
+[Vector3](#vector3)
+
+</td>
+<td>
+
+Scale factor to apply to the model along each axis
+
+</td>
+</tr>
+<tr>
+<td><code>color</code></td>
+<td>
+
+[Color](#color)
+
+</td>
+<td>
+
+Solid color to use for the whole model. If `use_embedded_materials` is true, this color is blended on top of the embedded material color.
+
+</td>
+</tr>
+<tr>
+<td><code>use_embedded_materials</code></td>
```

Comment:
```suggestion
<td><code>embedded_materials</code></td>
```

it's already a boolean

Reply:
Without the verb I'm a bit confused about what this might mean. "Has some embedded materials"? "Did embed materials"?

Reply:
🤷 we don't have verbs for any other bool flags. but maybe this one is justified to be different. I don't feel strongly about it but curious what others think.

Reply:
I think all our other bools can be interpreted as "is it X" or "does it X" and it makes sense, while this one is a bit different which is why it sounds weird to me. I also don't have strong feelings here though.

Reply:
Makes sense. Since neither of us feel strongly, @defunctzombie do you want to weigh in?

Reply:
The `use` feel superfluous. If an embedded_material is present it should be used. We can allow a topic setting to toggle it out-of-band of we want to provide a visualization option but the message doesn't need to convey that.

Reply:
> If an embedded_material is present it should be used.

well no, that's the whole point of this field. Gives the user a way to disable the usage of the embedded material. Hende `use_embedded_materials=false`.

Reply:
Is this something you need to configure when sending the message? You added the mesh - why not add one with no material? We don't override other properties of a mesh.

Is this somehow a very common thing to do? Seems like this field could be dropped?

Reply:
If this field needs to remain and is about enabling/disabled a property of the mesh I would favor using the words `enable` or `disable`

Reply:
> You added the mesh - why not add one with no material?

Perhaps you are not a 3D artist and someone just handed you a mesh to use. Or you want to use the same mesh in multiple ways, such as a regular-looking car and a "ghost car" at the same time. Just some ideas. I don't feel strongly that this is an important feature, just explaining my guess at why it exists.